content/en/about/feature-stages/index.md

```diff
@@ -55,12 +55,10 @@ Below is our list of existing features and their current phases. This informatio
 |-------------------|-------------------
 | [Prometheus Integration](/docs/tasks/observability/metrics/querying-metrics/) | Stable
 | [Local Logging (STDIO)](/docs/tasks/observability/mixer/logs/collecting-logs/) | Stable
-| [Statsd Integration](/docs/reference/config/policy-and-telemetry/adapters/statsd/) | Stable
 | [Client and Server Telemetry Reporting](/docs/reference/config/policy-and-telemetry/) | Stable
 | [Service Dashboard in Grafana](/docs/tasks/observability/metrics/using-istio-dashboard/) | Stable
 | [Istio Component Dashboard in Grafana](/docs/tasks/observability/metrics/using-istio-dashboard/) | Stable
```

Comment:
Maybe remove? This links to the same docs, and the dashboards have shifted a bit with the removal of all components except for Pilot.

Reply:
removed